### DISCUSSION TITLE
deepin-camera: Non customizeable output format

### DISCUSSION THREAD

**Discussion by barrulik** [linuxdeepin/developer-center#3421] [Features Request & Ideas | 特性请求 & 头脑风暴]
the app is great, but why cant we change the output format to mp4/mkv for videos or png for photos

**Comment by SeanRamey** [comment]
+1 to that.

**Comment by BLumia** [comment]
Hi, sorry for the late reply since we didn't check issues that were reported outside the developer-center repo. I'll help you move the issue there and see if we can add such feature to the project :)

**Reply by BLumia** [reply]
cc @meiyixiang
